src/test/java/tests/ChangePasswordTest.java

```diff
@@ -0,0 +1,35 @@
+package tests;
+
+import common.Constant;
+import helper.LogHelper;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_object.ChangePasswordPage;
+import page_object.LoginPage;
+
+public class ChangePasswordTest extends BaseTest {
+    ChangePasswordPage changePasswordPage = new ChangePasswordPage();
+    LoginPage loginPage = new LoginPage();
+
+    @Test(description = "User can't change password when New Password and Confirm Password are different.")
+    public void tc09_UserCantChangePasswordWhenNewPasswordAndConfirmPasswordAreDifferent() {
+        LogHelper.info("Click login tab");
+        loginPage.clickLoginTab();
+
+        LogHelper.info("Enter valid data and click Login button");
+        loginPage.login(Constant.USERNAME, Constant.PASSWORD);
+
+        LogHelper.info("Click change password tab");
+        changePasswordPage.clickChangePasswordTab();
+
+        LogHelper.info("Enter valid value into all fields");
+        changePasswordPage.changePassword(Constant.PASSWORD, "a123:\"/{}!@$\\", "b456:\"/{}!@$\\");
+
+        LogHelper.info("Get error message");
+        String actual = changePasswordPage.getErrorMessage();
```

Comment:
var name should be more specific.

Reply:
Enter valid information into "Current Password" textbox but enter "a123:"/{}!@$\" into "New Password" textbox and "b456:"/{}!@$\" into "Confirm Password" textbox. I took it from the test case so should I need to change it?

Reply:
Fixed. Please check again

Reply:
what did you fix?

Reply:
I fixed it. Please check again

Reply:
I saw that you changed on `"Change password with new password and confirm password are different"` step. It's not about that.
He told you at `"Get error message"` step.
The variables name are  just `actual` and `expected`.
`actual` and `expected` of what, you need to specify them